nx-libs.spec

```diff
@@ -586,7 +590,7 @@ rm -r %{buildroot}%{_includedir}/nx-X11/Xtrans
 %{_bindir}/nxproxy
 %{_datadir}/man/man1/nxproxy.1*
 %dir %{_libdir}/nx/bin
```

Comment:
Better change this to `%{_libexecdir}` as well, I guess.

Reply:
More information: according to the [Fedora Packaging Guidelines](https://fedoraproject.org/wiki/Packaging:Guidelines#Packaging_Static_Libraries), `.la` files should be deleted and not shipped - static libraries should be avoided at all costs.

The better handling would probably thus to be deleting these files in the `%install` section like we do in `debian/rules`.